plugins/modules/azure_rm_aks.py

```diff
@@ -973,6 +1014,9 @@ def create_network_profile_instance(self, network):
     def create_aad_profile_instance(self, aad):
         return self.managedcluster_models.ManagedClusterAADProfile(**aad) if aad else None
 
+    def create_api_server_access_profile_spec(self, server_access):
```

Comment:
The call does not match the name of function define
```suggestion
    def create_api_server_access_profile_instance(self, server_access):
```